Bugfixes for Beta

ilkober opened the pull request:
Fixed a misspelled tooltip for Portuguese MIO.
Corrected the arrest of Raghib Al-Nashashbi not working properly.

ilkober committed:
Tooltip + PAL retiring fix

Fixed a misspelled tooltip for Portuguese MIO.
Corrected the arrest of Raghib Al-Nashashbi not working properly.

ilkober committed:
Fixing some misspelled tokens

ilkober committed:
Removed controller_priority which is not applied properly

ilkober committed:
More bugfixes

One of the democratic Nigerian ideas will now be correctly applied.
Soviets will no longer be able to relocate the industry into the same state.
Fixed a wargoal for Scotland regarding the Celtic Liberation.

ilkober committed:
Operatives fixes + decision fix

[BftB] Integration of Turan lands now correctly grants Szekely, Kuwaiti, Kosovan and Macedonian operatives.
Fixed the "Monarchist debates in Sejm" decisions triggers.

marijn211 committed:
Fix to MolRib adjustment I did

marijn211 commented on common/national_focus/portugal.txt:
Why would you replace the tooltip of the idea with a tooltip of an MIO that might not even be loaded?

ilkober replied:
por_ogme does not exist, hence why I just picked the other available option

ilkober committed:
Finnish Charcter Fix + MIO check

Checking cores should work correctly for MIOs.
Fixed incorrect character checks for Finnish BoP

ilkober committed:
Merge branch 'beta-bugfixes-02-2024' of https://github.com/deliciousmods/1956_beta into beta-bugfixes-02-2024

ilkober committed:
Update portugal.txt

marijn211 committed:
Update portugal.txt

marijn211 committed:
Merge branch 'beta-bugfixes-02-2024' of https://github.com/deliciousmods/1956_beta into beta-bugfixes-02-2024

marijn211 committed:
Fixed error-spamming typo

marijn211 committed:
Update poland.txt

marijn211 committed:
Fixed broken and previously hidden CUB focus

ilkober committed:
Update r56_caribbean_shared.txt

ilkober committed:
Old content

marijn211 committed:
Hopefully stop Poland from misbehaving on hist

marijn211 committed:
A couple very small bugfixes

marijn211 committed:
Copied bugfix by Spicy so I can test hist

marijn211 reviewed (approved):
Looks good, testing one more bugfix then accepting

marijn211 committed:
Fixed Bulgaria not boosting opinion to unlock focus
